src/image-copy/azext_imagecopy/cli_utils.py

```diff
@@ -0,0 +1,44 @@
+import sys
+import json
+
+from subprocess import check_output, STDOUT, CalledProcessError
+from azure.cli.core.util import CLIError
+import azure.cli.core.azlogging as azlogging
+
+logger = azlogging.get_az_logger(__name__)
+
+
+def run_cli_command(cmd, return_as_json=False):
+    try:
+        cmd_output = check_output(cmd, stderr=STDOUT, universal_newlines=True)
+        logger.debug('command: %s ended with output: %s', cmd, cmd_output)
+
+        if return_as_json is True:
+            if cmd_output:
+                json_output = json.loads(cmd_output)
+                return json_output
+            else:
+                raise CLIError("Command returned an unexpected empty string.")
```

Comment:
Would be good to add which command to this error message?

Reply:
There's one later on - this raises an exception in the try-catch block and the catch prints the cmd